### DISCUSSION TITLE
5 Plugin Ideas 👍

### DISCUSSION THREAD

**Discussion by Looodon** [runelite/runelite#17366] [Ideas]
# Plugins ideas:
1: Jad fight practice for prayer flicking, a sound plugin (NOT A VISUAL SIMULATION)
_Play 2 different sounds randomly. Player has to click on the right prayer based on which sound it is. Maybe have feedback if you did it right or wrong, no in-game models used or simulated._

2: DPS calculator where you can compare 2 setups without wearing them and select on which monster
_Google sheets is not a plugin :D_

3: Show weight somewhere _at all times_, so you can quickly try outfits

4: Change ground item text size/opacity
_The Dynamic Overly Font "small" is not small enough. When there so many drops of the floor I cant see the small monsters I'm fighting or what I'm clicking on_

> EDIT: Removed 1 of the suggestions of the bank plugin i was looking for, added descriptions

**Comment by DominickCobb-rs** [comment]
1. https://www.reddit.com/r/2007scape/comments/wiq7dv/jad_simulator_public_release_a_trainer_i_made_in/
2. https://docs.google.com/spreadsheets/d/1wBXIlvAmqoQpu5u9XBfD4B0PW7D8owyO_CnRDiTHBKQ/edit
3. 
![Plugins](https://github.com/runelite/runelite/assets/138266282/13b8a474-23c6-4709-86db-1e20eaca3d0a)
4. 
![Mouse over run energy](https://github.com/runelite/runelite/assets/138266282/e9152e65-b3c2-4bf9-8573-5cd97bb96bcc)
5. 
![Runelite plugin. Not perfect but close enough.](https://github.com/runelite/runelite/assets/138266282/eb3c5fbe-683d-43be-85d2-5596bcf9bb88)

**Reply by Looodon** [reply]
Hey,

1-2) I've used 1 & 2 since forever, my hope was that they'd be in the client as plugins for easier/faster access
3) Number 3 worked as I wished thanks :D 
4) I was hoping it would be shown at all times as you equip and unequip things
5) Sadly, I had the Dynamic Overly on small this whole time but its still big, when there so many drops of the floor i cant see the small monsters i'm fighting or what i'm clicking on

I appreciate the time to share these with me and for your time ^_^

**Comment by JebScape** [comment]
Jagex is never going to allow a boss simulator. It’s basically cheating.

**Reply by Looodon** [reply]
What would not be allowed?
The plugin's function would be:
Step 1)
Play 1 of 2 different sounds, but not the ones from jad for practice (the metronome plugin makes sounds)

Step 2)
The player has to click on the right prayer based on which sound it is

Maybe:
-The prayer gets highlighted for the appropriate sound
-There would be feedback if u click the wrong one
-There is no visual added, its just a sound and a possibly a prayer highlighted

**Reply by DominickCobb-rs** [reply]
At that point why wouldn't you just use the two things I linked? It's impractical to put into the client at all, and you would learn nothing from doing things that aren't the content you're trying to learn.

**Reply by Looodon** [reply]
If by two things you mean DPS calc & Jad sim, I did use them. let me simplify it for you 

1) The DPS excel sheet where I have to minimize the game, have a google account, log in, and make sure it dont mess up the cells is a helpful tool that I use regularly. Unfortunately, google sheets is not a Runelite plugin, which my suggestion was for. It does not exist as a plugin and so I suggested it. Most plugins don't need to exist, but people are free to use, ask for, or make them.

2) The Jad sim you sent me is like practicing in a different game. If you don't like my suggestions you don't need to be argue with me about my personal preference. No one is forcing you to make them lol

**Reply by Nightfirecat** [reply]
Jagex has asked that other simulated content (such as a simulated leviathan fight in the quest helper plugin) be removed in the past, so we are not interested in trying to add anything else of that nature at this time.

**Reply by Looodon** [reply]
I don't know if you read how I described it, even though Jad is not being simulated at all in any of this. One sound plays then another plays and you try to click on the prayers yourself to practice. is it considered a simulation if there is a plugin that plays one of 2 sounds randomly, and that's it? 

Extra: I suggested the _**possibility**_ in an inquisitive intent if the prayers are highlighted for you and there is feedback if u click the wrong one, but if that specific part is not allowed then that's fine.
